server/routes/blackjack.py

```diff
@@ -36,9 +65,29 @@ def play_blackjack():
     except json.decoder.JSONDecodeError:
         return {"error": "Malformed request"}, 400
 
-@blackjack_bp.route("/api/blackjack/start", methods=["GET"])
+@blackjack_bp.route("/api/blackjack/checkfunds", methods=["POST"])
+def check_funds_blackjack():
+    try:
+        data = json.loads(request.data)
+        bet_amount = data["amount"]
+        total_tickets = (
+            db.session.query(func.sum(Transaction.ticket_amount))
+            .filter(Transaction.user_id == session["user_id"])
+            .scalar()
+        )
+        if bet_amount > total_tickets:
+            return {
+                "success": False,
+                "message": "You are wagering more tickets than you currently have"
+            }
+        return {"success": True}
+    except json.decoder.JSONDecodeError:
+        return {"error": "Malformed request"}, 400
```

Comment:
Add lines between methods

Reply:
By methods do you mean line 83/84? or the return content?

Reply:
line break after line 85. New methods starts line 86.